core/Pos/Binary/Core/Types.hs

```diff
@@ -17,57 +17,53 @@ import qualified Pos.Data.Attributes     as A
 -- verbosity and clarity
 
 instance Bi T.Timestamp where
-    get = label "Timestamp" $ fromInteger <$> get
+    get = {-# SCC get_Timestamp #-} label "Timestamp" $ fromInteger <$> get
     put = put . toInteger
 
 instance Bi T.EpochIndex where
-    get = label "EpochIndex" $ T.EpochIndex . getUnsignedVarInt <$> get
+    get = {-# SCC get_EpochIndex #-} label "EpochIndex" $ T.EpochIndex . getUnsignedVarInt <$> get
     put (T.EpochIndex c) = put (UnsignedVarInt c)
 
 instance Bi (A.Attributes ()) where
-    get = label "Attributes" $
+    get = {-# SCC get_Attributes #-} label "Attributes" $
         A.getAttributes (\_ () -> Nothing) (Just (128 * 1024 * 1024)) ()
     put = A.putAttributes (\() -> [])
 
 instance Bi T.Coin where
     put = mapM_ putWord8 . BinCoin.encode
-    get = label "Coin" $ BinCoin.decode
+    get = {-# SCC get_Coin #-} label "Coin" $ BinCoin.decode
 
 instance Bi T.CoinPortion where
     put = put . T.getCoinPortion
-    get = label "CoinPortion" $ get >>= T.mkCoinPortion
+    get = {-# SCC get_CoinPortion #-} label "CoinPortion" $ get >>= T.mkCoinPortion
 
 instance Bi T.LocalSlotIndex where
-    get = label "LocalSlotIndex" $ T.LocalSlotIndex . getUnsignedVarInt <$> get
+    get = {-# SCC get_LocalSlotIndex #-} label "LocalSlotIndex" $ T.LocalSlotIndex . getUnsignedVarInt <$> get
     put (T.LocalSlotIndex c) = put (UnsignedVarInt c)
 
 instance Bi T.SlotId where
     put (T.SlotId e s) = put e >> put s
-    get = label "SlotId" $ do
+    get = {-# SCC get_SlotId #-} label "SlotId" $ do
         siEpoch <- get
         siSlot <- get
         let errMsg =
                 formatToString ("get@SlotId: invalid slotId ("%int%")") siSlot
         unless (inRange (0, epochSlots - 1) siSlot) $ fail errMsg
         return $ T.SlotId {..}
 
-instance Bi T.EpochOrSlot where
```

Comment:
Basically, I deleted all `Bi` instances that were added in `15f83a228c7798a4a612c69b6d62a5335c2bacf7` because they were added for `GtGlobalState` that is now serialized with `Store`.

They serve no other purpose.